docs/tutorials/intro.md

```diff
@@ -106,45 +110,54 @@ fn main() {
 }
 ```
 
-In this case, the parsing is performed at compilation time, so we don't need to handle an error case. Try passing an malformed identifier, like "foo-bar" and try to call `cargo check`.
+In this case, the parsing is performed at compilation time, so we don't need to handle an error case. Try passing an malformed identifier, like "foo-bar" and call `cargo check`.
 
-*Notice:* The compile time macros `langid!`, and `locale!` don't support variants or extension tags, as storing these requires allocation. If you have such a tag you need to use runtime parsing.
+*Notice:* `locale!` does not support variants or extension tags (e.g. `u-nu-latn`), as storing these requires allocation. If you have such a tag you need to use runtime parsing.
```

Comment:
Issue: This statement is outdated. I suggest removing it. The restrictions on what you can and can't put in `locale!` are a detail that is better discovered from docs.rs.